ui/perfherder/graphs/LegendCard.jsx

```diff
@@ -29,10 +30,18 @@ const LegendCard = ({
         if (isVisible && newColors.length) {
           item.color = newColors.pop();
           item.visible = isVisible;
+          item.data = item.data.map(test => {
```

Comment:
I think this could also be:

    item.data = item.data.map(test => ({
        ...test, z: item.color[1],
    });

Reply:
That's a bit cleaner - thanks!